pkgs/top-level/python-packages.nix

```diff
@@ -8556,6 +8556,25 @@ in {
     };
   };
 
+  python-openid = buildPythonPackage rec {
+    name = "python-openid-${version}";
+    version = "2.2.5";
+
+    # use python3-openid instead
+    disabled = isPy3k;
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/p/python-openid/${name}.tar.gz";
+      sha256 = "1vvhxlghjan01snfdc4k7ykd80vkyjgizwgg9bncnin8rqz1ricj";
+    };
+
+    meta = {
+      description = "OpenID support for servers and consumers";
+      homepage = "https://github.com/openid/python-openid";
+      license = licenses.asl20;
```

Comment:
Of all packages I've contributed, this is probably the one I'm least suitable to maintain...